modal/cli/launch.py

```diff
@@ -22,15 +22,22 @@
     no_args_is_help=True,
     help="""
     Open a serverless app instance on Modal.
-
-    This command is in preview and may change in the future.
+    >⚠️  modal launch is experimental and may change in the future.
     """,
 )
 
 
 def _launch_program(
     name: str, filename: str, detach: bool, args: dict[str, Any], *, description: Optional[str] = None
 ) -> None:
+    console = make_console()
+    console.print(
+        rich.panel.Panel(
+            f"⚠️  modal launch {name} is experimental and may change in the future",
+        ),
+        style="bold yellow",
```

Comment:
Just being fussy but could we match the other warning boxes we show (yellow outline, regular weight white text).

Reply:
May also help to "code format" the CLI component.

Reply:
Adjusted:

![Screenshot 2025-08-08 at 1 50 58 PM](https://github.com/user-attachments/assets/2b178dda-3b18-4159-a4ee-c9926d42bc0e)